src/mn_setup.c

```diff
@@ -1438,6 +1438,9 @@ static setup_menu_t stat_settings2[] = {
 
     {"Use-Button Timer", S_ONOFF, M_X, M_SPC, {"hud_time_use"}},
 
+    {"Command Display", S_ONOFF | S_STRICT, M_X, M_SPC, {"hud_command_display"},
```

Comment:
How about "Input History"?

Reply:
"Command Display" has been the established name for a couple years now, but something like "Show Input Commands" or your suggestion is probably more intuitive.

Reply:
Yes, please.

Reply:
Let's go with "Show Command History". Fixed by https://github.com/fabiangreffrath/woof/pull/1692/commits/52d1689d1833160ae80bddbe1d4fe44c1d7407a4